demisto_sdk/commands/test_content/xsiam_tools/xsiam_client.py

```diff
@@ -126,6 +128,22 @@ def _session(self) -> requests.Session:
     def _session(self, value: requests.Session):
         self.__session = value
 
+    @lru_cache
+    @retry(times=5, exceptions=(RuntimeError))
```

Comment:
@GuyAfik Is this good enough?

Reply:
i'd also add Connection / Timeout exceptions (from requests module)

Reply:
I added the **ConnectionError** and **Timeout** exceptions.